relnotes/makelayout.feature.md

```diff
@@ -0,0 +1,37 @@
+* `ocean.util.log.Config`
+
+    The `configureLogger` and `configureOldLoggers` functions
+    have gained an additional `makeLayout` parameter.
+    This parameter is used to instantiate a `Layout` from a name, allowing
+    users to have custom layout usable from their config files.
+    For example, a config entry would be:
+    ```
+    [LOG.myapp.supermodule]
+    level = info
+    file  = log/supermodule.log
+    file_layout = aquatic
+    console_layout = submarine
+    ```
+    And calling `configureOldLoggers` should be done as such:
+    ```
+    void myConfigureLoggers (
+        ClassIterator!(Config, ConfigParser) config,
+        MetaConfig m_config,
+        Appender delegate ( istring file, Layout layout ) file_appender,
+        bool use_insert_appender = false)
+    {
+        scope makeLayout = (cstring name)
```

Comment:
I don't think this is correct. What would be the type of `makeLayout`? It might work for D2 though